### ISSUE TITLE
Not work for Kali VM

### ISSUE THREAD

**Issue by ImJoke** [milesrichardson/docker-onion-nmap#1]
I was trying to use this tool and it worked, but only for raspberry pi while for Kali Linux virtual machine did not work

**Comment by milesrichardson** [comment]
yeah this thing is super outdated and doesn't work atm. I've been meaning to fix it, I think it just needs a few upgrades

**Comment by chrisbensch** [comment]
I've been trying to get this to work, no matter what I try dnsmasq has issues with "address already in use" and "port already in use" scenarios.  I can't seem to get around it.

**Comment by chrisbensch** [comment]
> I've been trying to get this to work, no matter what I try dnsmasq has issues with "address already in use" and "port already in use" scenarios. I can't seem to get around it.

So I've removed dnsmasq altogether and recreated the same function (seems to be anyway) with unbound.  The image built will be 'chrisbensch/docker-onion-nmap:latest' once I upload today.  I also forked this repo with my changes so anyone can see what I did.

**Comment by chrisbensch** [comment]
> > I've been trying to get this to work, no matter what I try dnsmasq has issues with "address already in use" and "port already in use" scenarios. I can't seem to get around it.
> 
> So I've removed dnsmasq altogether and recreated the same function (seems to be anyway) with unbound. The image built will be 'chrisbensch/docker-onion-nmap:latest' once I upload today. I also forked this repo with my changes so anyone can see what I did.

I forgot to mention, I was having issues connecting to tor so I added 'lyrebird' for use as the obfs4 proxy and a couple of bridges from the torproject.
